app/models/prison.rb

```diff
@@ -6,6 +6,8 @@ class Prison < ActiveRecord::Base
   validates :estate, :name, :nomis_id, :slot_details, presence: true
   validates :enabled, inclusion: { in: [true, false] }
 
+  alias_attribute :email, :email_address
```

Comment:
I've specifically been trying to use `email_address` everywhere to avoid the ambiguity of email-as-address and email-as-document, so please don't do this.